packages/data-broker/src/services/indicator/IndicatorService.js

```diff
@@ -21,10 +21,16 @@ export class IndicatorService extends Service {
   }
 
   async pull(dataSources, type, options) {
-    if (type === this.dataSourceTypes.DATA_GROUP) {
-      throw new Error('Event pulling is not supported in IndicatorService');
+    switch (type) {
```

Comment:
I think this is the only service that needs to be updated to handle `syncGroup`

`DhisService` and `TupaiaService` both map the data source type to functions so should just error on a newly added type. (There's already precedence for this with `DhisService` only having one metadata puller);

`WeatherService` checks directly if type is `DATA_ELEMENT` or `DATA_GROUP` so will error if the type doesn't match.